### PR TITLE
TGP-933 Navigation for create a TGP record Journey

### PR DESCRIPTION
**Pull request by yohan-oc**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -92,10 +92,10 @@ POST        /create-record/goods-description-question/check  controllers.UseTrad
 GET         /create-record/cya-create-record                 controllers.CyaCreateRecordController.onPageLoad
 POST        /create-record/cya-create-record                 controllers.CyaCreateRecordController.onSubmit
 
-GET         /trader-reference                                controllers.TraderReferenceController.onPageLoad(mode: Mode = NormalMode)
-POST        /trader-reference                                controllers.TraderReferenceController.onSubmit(mode: Mode = NormalMode)
-GET         /trader-reference/check                          controllers.TraderReferenceController.onPageLoad(mode: Mode = CheckMode)
-POST        /trader-reference/check                          controllers.TraderReferenceController.onSubmit(mode: Mode = CheckMode)
+GET         /create/trader-reference                         controllers.TraderReferenceController.onPageLoad(mode: Mode = NormalMode)
```
Comment (besscerule) [inline conf/app.routes:95]: shouldn't this be create-record instead of create?

**Review by besscerule — CHANGES_REQUESTED** [review]

File: conf/app.routes (hunk 1 above)
Comment (yohan-oc) [reply conf/app.routes:95]: I raised this with Amaka. She asked me to stick what confluence has for now and same in the ticket too https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=861962859 :(

**Review by besscerule — DISMISSED** [review]

File: app/views/HasCorrectGoodsView.scala.html [hunk 2]
```diff
@@ -33,7 +33,7 @@ <h1 class="govuk-heading-l">@messages("hasCorrectGoods.heading", commodity.commo
     <p class="govuk-body">@messages("hasCorrectGoods.p1", commodity.description)</p>
 
     <p class="govuk-body">
-        <a href="https://www.trade-tariff.service.gov.uk/find_commodity" class="govuk-link" >@messages("hasCorrectGoods.p2.linkText")</a>.
+        <a href="https://www.trade-tariff.service.gov.uk/search?q=@commodity.commodityCode" class="govuk-link" >@messages("hasCorrectGoods.p2.linkText")</a>.
```
Comment (vinoocs) [inline app/views/HasCorrectGoodsView.scala.html:36]: In three amigos, Simon said the url should be https://www.trade-tariff.service.gov.uk/xi/commodities/commodityCode ie., https://www.trade-tariff.service.gov.uk/xi/commodities/0702000007

**Review by Ewan-James-Donovan — DISMISSED** [review]

**Review by vinoocs — DISMISSED** [review]

File: app/controllers/CyaCreateRecordController.scala [hunk 3]
```diff
@@ -59,7 +59,7 @@ class CyaCreateRecordController @Inject() (
 
   // TODO redirect to correct location and submit data
   def onSubmit(): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    Redirect(routes.IndexController.onPageLoad)
+    Redirect(routes.CreateRecordSuccessController.onPageLoad().url)
```
Comment (ChrisCookOC) [inline app/controllers/CyaCreateRecordController.scala:61]: Can we now remove the //TODO redirect to correct location above?

Comment (yohan-oc) [reply app/controllers/CyaCreateRecordController.scala:61]: This is done :)

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**